components/theme.ts

```diff
@@ -36,30 +38,26 @@ export interface Theme {
   others: T.Dict<string>;
 }
 
-const defaultTheme: Theme = {
-  colors: {},
-  sizes: {},
-  rawSizes: {},
-  fonts: {},
-  zIndexes: {},
-  others: {}
-};
+export interface ThemeProps {
```

Comment:
I guess in this case indeed it doesn't make much sens to have this type. This is because we use the mixin conditionally. But for another component where you would use a mixin without condition juste like this: `${myMixin}` it's easier to have the type available, because I don't want to have to write `${ props => myMixin(props.theme)}` in this case.

So I think we should keep the theme and try keep a consistent way of writing mixins.

Reply:
Fair point. Then I think `ThemeProps` is misleading. It is not the props of theme, it a props with a theme. What about `StyledProps` or `ThemedProps` ?

Reply:
Indeed, I'll rename `ThemedProps`